aws/cognito_user_pool/main.tf

```diff
@@ -126,6 +126,24 @@ resource "aws_cognito_user_pool" "this" {
       kms_key_id                     = lambda_config.value.kms_key_id
     }
   }
+
+  # This is based on
+  # https://github.com/lgallard/terraform-aws-cognito-user-pool/blob/master/main.tf
+  #
+  # Note that this implementation will work for configuring boolean and datetime
+  # attributes, but will not work for string or number attributes because those
+  # require constraint configurations in the form of additional nested dynamic
+  # blocks.
+  dynamic "schema" {
+    for_each = var.schemas
+    content {
```

Comment:
nit: for consistency with other blocks, add a newline between the `for_each` and `content` lines